### PR TITLE
Adding solutions to 7-kata

### PR DESCRIPTION
**Pull request by Maksym637**

### CODE REVIEW

**Comment by Maksym637** [comment]
Please, check solutions to kata 7.
Thanks in advance !

File: src/main/java/com/org/ita/Main.java [hunk 1]
```diff
@@ -5,5 +5,3 @@ public static void main(String[] args) {
         System.out.println("Run!");
     }
 }
-
-
```
Comment (lhalam) [inline src/main/java/com/org/ita/Main.java]: delete these changes

**Review by lhalam — COMMENTED** [review]
delete changes in Main.java

File: src/main/java/com/org/ita/Main.java (hunk 1 above)
Comment (Maksym637) [reply src/main/java/com/org/ita/Main.java]: I fixed it

**Pull request merged**